sos/report/__init__.py

```diff
@@ -1058,8 +1061,12 @@ def list_plugins(self):
                 if tmpopt is None:
                     tmpopt = 0
 
-                self.ui_log.info(" %-25s %-15s %s" % (
-                    opt.plugin + "." + opt.name, tmpopt, opt.desc))
+                self.ui_log.info(
+                    (
+                        " %-25s %-15s %s"
+                        % (f"{opt.plugin}.{opt.name}", tmpopt, opt.desc)
+                    )
+                )
```

Comment:
Same here - got the "inner" substitution, but we can replace the spacing as well.